### PR TITLE
VA-5 include shared style

### PR DESCRIPTION
**Pull request by alirni**
VA-5

### CODE REVIEW

**Review by salmanclever — APPROVED** [review]

**Review by salmanclever — APPROVED** [review]

File: src/box-profile.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<link rel="import" href="../bower_components/polymer/polymer-element.html" />
+<link rel="import" href="../bower_components/iron-icon/iron-icon.html" />
+<link rel="import" href="app-icons.html" />
+
+<dom-module id="box-profile">
+  <template>
+    <style>
+
+    .profile-img {
+    /*background-color: #fff;*/
```
Comment (alirni) [inline src/box-profile.html]: fix tab

File: src/box-menu-icon.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    ready () {
+      super.ready();
+      // this.$.one.addEventListener('fireevent1', this.myEvent);
+      // this.$.two.addEventListener('fireevent2', this.myEvent);
+      // this.$.three.addEventListener('fireevent3', this.myEvent);
+    }
+
+    fireevent () {
```
Comment (alirni) [inline src/box-menu-icon.js]: https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent

File: src/box-profile.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    :host .container {
+      @apply --box-profile;
+    }
+
+    </style>
+
+    <div align="right" class="container">
+      <div>
+        <img src="/../images/profile-pic.png" class="profile-img"/>
```
Comment (alirni) [inline src/box-profile.html]: mast change to iron-image

File: src/volcano-app.html [hunk 4]
```diff
@@ -1,25 +1,33 @@
...
+<link rel="import" href="box-breadcrumb.html" />
+<link rel="import" href="box-menu-icon.html" />
+<link rel="import" href="page-view1.html" />
+<link rel="import" href="page-view2.html" />
+<link rel="import" href="page-view3.html" />
+<link rel="import" href="page-404.html" />
 
 <dom-module id="volcano-app">
-  <template>
+    <template>
```
Comment (alirni) [inline src/volcano-app.html:30]: fix tab

File: src/volcano-app.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-(function (){
+(function() {
```
Comment (alirni) [inline src/volcano-app.js:1]: fix indent

**Pull request closed without merging**